### ISSUE TITLE
Pretty sure extra steps are required to properly port forward with ProtonVPN

### ISSUE THREAD

**Issue by Seiru** [Chillsmeit/qBittorrent-ProtonVPN-Guide#3]
Was confused why qBittorrent was still saying "firewalled" after following the guide, and after a some research I stumbled onto the following:

https://github.com/qdm12/gluetun/issues/1751
https://github.com/t-anc/GSP-Qbittorent-Gluetun-sync-port-mod

After adding that mod with the proper settings, qBittorrent shows as the connection being fine.

Thanks for getting me 99% there though, I was totally in the dark before finding this guide.

**Comment by CreepyGiantz** [comment]
How did you set up that mod? I find the documentation of the thread a tad bit confusing

**Comment by ilGimmy** [comment]
I agree with @Seiru ; ProtonVPN documentation is clear about giving a random port for port forwarding: https://protonvpn.com/support/port-forwarding-manual-setup?srsltid=AfmBOor1rCpdBwsLPy4wWG1zcsgmLwvkHYjDqfWCFvExAq-_FTxVfM9v#linux
